src/cc2olx/models.py

```diff
@@ -272,6 +273,25 @@ def get_resource_content(self, identifier):
                     )
                     raise
                 return "html", {"html": html}
+            elif 'web_resources' in str(res_filename):
```

Comment:
You don't have to cast `res_filename` always to string. The reason it's done in some places is that `res_filename` is of `pathlib.Path` type. Some modules from python3.5 accept only string path, while we need to support both 3.5 and 3.8, so that's only for backward compatibility.

In this case `'web_resources' in res_filename` should work fine as well as for string formatting below.

Reply:
@0x29a I tried removing string conversion from above condition but it failed when I ran tests indicating `TypeError: argument of type 'PosixPath' is not iterable`.
Can you please suggest where it is going wrong if it is not the expected behaviour.

Reply:
@RiteshChouhan ah, sorry, I meant `imghdr.what(str(res_filename))`, but looks like in python3.5 it's also doesn't expect `PathLike` object. So it's ok to leave this as is :+1: